pkg/crc/constants/constants_darwin.go

```diff
@@ -8,5 +8,6 @@ const (
 )
 
 var (
-	TrayBinaryPath = filepath.Join(CrcBinDir, TrayBinaryName)
+	TrayAppBundlePath = filepath.Join(CrcBinDir, TrayBinaryName)
+	TrayBinaryPath    = filepath.Join(TrayAppBundlePath, "Contents", "MacOS", "CodeReady Containers")
```

Comment:
`TrayBinaryPath` is not being used at the moment, seems like an oversight.

Reply:
This is needed in the trayPlistConfig, but am again forming this path there, should use the constant.